assets/src/editor/events/tickets/editor-ticket/use-tickets-registration-count.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * useTicketsRegistrationCount
+ *
+ * A custom react hook for retrieving the ticket to
+ * registration count mapping.
+ *
+ * @return {Object} A mapping from ticket ID to registration count;
+ */
+const useTicketsRegistrationCount = () => {
+	const { tktRegCount = {} } = window.eeEditorEventData;
```

Comment:
Shouldn't we use selectors for retrieval?
My understanding based on React docs is that hooks are more oriented for:
`They let you use state and other React features without writing a class.`
Also, selectors can be composed.

Reply:
cc @tn3rb

Reply:
In `reselect` docs there is an example of how we might benefit from composing selectors: https://github.com/reduxjs/reselect

Reply:
`window.eeEditorEventData` is the var I used to dump all of our entity data into the DOM.
It is not a redux store or anything, it is just a big object.

Reply:
I created that hook just in case we change the way that registration count is retrieved. Currently it's retrieved from DOM dump.